### DISCUSSION TITLE
针对评分弹幕的屏蔽

### DISCUSSION THREAD

**Discussion by varae24** [the1812/Bilibili-Evolved#4493] [功能建议 / Ideas]
目前已有的【删除视频弹窗】很好地改善了我的视频观看体验，但评分弹窗所导致的诸如“1、2……”的弹幕却仍然会在视频中存在，希望可以推出针对此类弹幕的屏蔽功能，感激不尽

**Comment by Geeyun-JY3** [comment]
b 站自带的屏蔽弹幕功能做不到吗？直接添加屏蔽词不就完了……

**Reply by varae24** [reply]
因为直接添加“1”、“2”之类的纯数字屏蔽词的话可能会误伤到其他较好的弹幕，可以的话希望能仅屏蔽这类特殊弹幕.

**Reply by Geeyun-JY3** [reply]
说错了，纯数字弹幕应该用正则屏蔽。

**Reply by varae24** [reply]
学到了，才发现这个功能(●'◡'●)

**Comment by hexadecimal233** [comment]
可以试用一下pakku这个插件，可以合并弹幕

**Reply by varae24** [reply]
好的，我去试试
